dataland-frontend/tests/component/components/pages/DatasetOverview.cy.ts

```diff
@@ -87,6 +75,7 @@ describe("Component tests for the DatasetOverview page", () => {
       keycloak: keycloakMock,
     }).then((mounted) => {
       validateTabBar(0, keycloakMock);
+      cy.wait(100);
```

Comment:
wait statements in component tests sollten eigentlich nicht notwendig sein

Reply:
entfernt